server/bytestream/bytestream_test.go

```diff
@@ -62,3 +68,35 @@ func TestManifest_TooManyFilesZip(t *testing.T) {
 	assert.Nil(t, manifest)
 	assert.Contains(t, err.Error(), "code = Unimplemented")
 }
+
+func createBytestreamer(b []byte) Bytestreamer {
+	return func(ctx context.Context, env environment.Env, url *url.URL, offset int64, limit int64, writer io.Writer) error {
+		log.Printf("%d %d", offset, limit-offset)
+		writer.Write(b[offset : offset+limit])
+		return nil
+	}
+}
+
+func validateZipContents(t *testing.T, entry *zipb.ManifestEntry, expectedContent string, streamer Bytestreamer) {
+	var buf bytes.Buffer
+	streamSingleFileFromBytestreamZipInternal(nil, nil, nil, entry, &buf, streamer)
+	out := make([]byte, entry.GetUncompressedSize())
+	_, err := io.ReadFull(&buf, out)
+	assert.Nil(t, err)
```

Comment:
there is also a "require" package instead of "assert" if you need to fail the test as soon as a precondition is false

Reply:
I simplified these and dropped the dep on the asserts package so that i could feel cool / like a Gopher or whatever.